src/main/java/net/sf/mzmine/modules/peaklistmethods/alignment/adap3/ADAP3AlignerTask.java

```diff
@@ -0,0 +1,557 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package net.sf.mzmine.modules.peaklistmethods.alignment.adap3;
+
+import dulab.adap.common.algorithms.machineleanring.OptimizationParameters;
+import dulab.adap.datamodel.Component;
+import dulab.adap.datamodel.Peak;
+import dulab.adap.datamodel.PeakInfo;
+import dulab.adap.datamodel.Project;
+import dulab.adap.datamodel.ReferenceComponent;
+import dulab.adap.datamodel.Sample;
+import dulab.adap.workflow.AlignmentParameters;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Map.Entry;
+import java.util.NavigableMap;
+import java.util.TreeMap;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import net.sf.mzmine.datamodel.DataPoint;
+import net.sf.mzmine.datamodel.Feature;
+import net.sf.mzmine.datamodel.IsotopePattern;
+import net.sf.mzmine.datamodel.MZmineProject;
+import net.sf.mzmine.datamodel.PeakIdentity;
+import net.sf.mzmine.datamodel.PeakList;
+import net.sf.mzmine.datamodel.PeakListRow;
+import net.sf.mzmine.datamodel.RawDataFile;
+import net.sf.mzmine.datamodel.impl.SimplePeakInformation;
+import net.sf.mzmine.datamodel.impl.SimplePeakList;
+import net.sf.mzmine.datamodel.impl.SimplePeakListRow;
+import net.sf.mzmine.modules.peaklistmethods.qualityparameters.QualityParameters;
+import net.sf.mzmine.parameters.ParameterSet;
+import net.sf.mzmine.taskcontrol.AbstractTask;
+import net.sf.mzmine.taskcontrol.TaskStatus;
+
+/**
+ *
+ * @author aleksandrsmirnov
+ */
+public class ADAP3AlignerTask extends AbstractTask {
+    
+    private static final Logger LOG = Logger.getLogger(
+            ADAP3AlignerTask.class.getName());
+    
+    private static final double MZ_TOLERANCE = 0.1;
```

Comment:
There should not be such a hard-coded constant here. You already have the MZ_RANGE parameter.